pkg/chartverifier/checks/charttesting.go

```diff
@@ -411,10 +412,13 @@ func installAndTestChartRelease(
 	return result
 }
 
-func setOCVersion(holder AnnotationHolder, versioner Versioner) error {
-	// oc.GetVersion() returns an error both in case the oc command can't be executed and
+func setOCVersion(opts *CheckOptions, versioner Versioner) error {
+	holder := opts.AnnotationHolder
+	settings := opts.HelmEnvSettings
```

Comment:
I would add `holder` and `settings` as arguments to this function instead of depending on a whole `CheckOptions` value to be present.

Reply:
Updated.